templates/clientheaders.any.j2

```diff
@@ -1 +1 @@
-"*"
\ No newline at end of file
+"*"

```

Comment:
???

Reply:
This is a part of Dispatcher's config. This file can be corrected manually after Dispatcher's installation.